src/analysis_and_optimization/Optimize.ml

```diff
@@ -1281,7 +1333,8 @@ let settings_const b =
   ; lazy_code_motion= b
   ; optimize_ad_levels= b
   ; preserve_stability= not b
-  ; optimize_soa= b }
+  ; optimize_soa= b
+  ; optimize_opencl= false }
```

Comment:
I think we'd want this to be enabled as part of `all_optimizations` so this should be `b`

Reply:
idk, personally this is a very specific optimization for a piece of hardware. I'd rather the user explicitly flips it on